Source/FakeItEasy/Creation/DefaultFakeCreatorFacade.cs

```diff
@@ -126,6 +126,11 @@ public IFakeOptionsBuilder<T> Implements(Type interfaceType)
                 return this;
             }
 
+            public IFakeOptionsBuilder<T> Implements<TInterface>()
+            {
+                return Implements(typeof (TInterface));

```

Comment:
You're very accommodating.

Yeah, for number 3, it was the space that was the problem. I'd lost my backticks when copying my comment and didn't notice.
Not sure about VS adding it. For me it doesn't, but that could be due to my VS configuration or the fact that I have ReSharper (and its configuration, I suppose). Dealing with various projects' conventions is a pain, no?
The warnings emitted by the compilation can be your guide—FakeItEasy usually builds without warnings.